node/src/GlideClient.ts

```diff
@@ -258,7 +258,7 @@ export class GlideClient extends BaseClient {
     }
 
     /** Get the name of the primary's connection.
-     *  See https://valkey.io/commands/client-getname/ for more details.
+     * @see {@link https://valkey.io/commands/client-getname/|valkey.io} for more details.
```

Comment:
Somewhere we have
> See \<link> for details.

But somewhere
> See \<link> for more details.

Please align.

Reply:
Either is fine.